pyuvdata/uvdata.py

```diff
@@ -1719,6 +1683,21 @@ def read_miriad(self, filepath, correct_lat_lon=True, run_check=True,
                 Ex: ['xx', 'yy', ...]
             time_range: len-2 list containing min and max range of times (Julian Date) to read-in.
                 Ex: [2458115.20, 2458115.40]
+            read_data: Read in the visibility and flag data. If set to false,
+                only the basic header info and metadata (if read_metadata is True)
```

Comment:
Same as above, no `read_metadata` kwarg.

Reply:
Same reasoning as above, your HDF5 files only seem to have 2 levels.